src/bp-groups/bp-groups-functions.php

```diff
@@ -3844,7 +3844,11 @@ function bp_group_type_short_code_add_body_class( $class ) {
 		$class[] = 'groups';
 		$class[] = 'buddypress';
 		$class[] = 'buddyboss';
-		$class[] = 'bb-buddypanel';
+		/**
+		 *This class commented because this class will add when buddypanel enable
+		 *and this condition already in the theme
+		 */
+		//$class[] = 'bb-buddypanel';
```

Comment:
@mahedihasannoman @dipskakadiya 

We can not remove this class as so many CSS is written into the theme based on the class.
So, You can make it conditional here OR create PR for the theme to add this class from the theme itself.

Reply:
@KartikSuthar , We are already adding this class “bb-buddypanel” from theme conditionally. https://nimb.ws/ZSniPA